components/blocks/Landing/Landing.test.ts

```diff
@@ -0,0 +1,20 @@
+import { mountSuspended } from '@nuxt/test-utils/runtime'
+import Landing from './Landing.vue'
+
+describe('Landing Component', () => {
+  const games = [
+    { name: 'Getting Over It With Bennet Foddy' },
+    { name: 'Getting Over It With Bennet Foddy' },
+    { name: 'Getting Over It With Bennet Foddy' },
+  ]
+
+  it('should render without crashing', async () => {
+    const wrapper = await mountSuspended(Landing, {
+      props: {
+        games,
+      },
```

Comment:
These props aren't actually used it seems. I think this will also actually call the backend because you haven't mocked getLeaderboards here.